tests/csapi/apidoc_room_state_test.go

```diff
@@ -195,5 +195,129 @@ func TestRoomState(t *testing.T) {
 				},
 			})
 		})
+		// sytest: POST /rooms/:room_id/state/m.room.name sets name
+		t.Run("POST /rooms/:room_id/state/m.room.name sets name", func(t *testing.T) {
```

Comment:
It's actually a PUT here, not POST. Please correct the test name (though the sytest comment remains unchanged).